<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="file_qrc.qrc">
    <normaloff>:/icon/system_monitor.ico</normaloff>:/icon/system_monitor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 26, 46);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_error">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 69, 96);
border-radius: 3px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_error">
            <property name="text">
             <string>Error</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close_popup">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border-radius: 3px;
font: 75 10pt &quot;Ubuntu&quot;;
}

QPushButton:hover {
color: rgb(255, 255, 255);
background-color: rgb(26, 26, 46);
}

QPushButton:pressed {
color: rgb(255, 255, 255);
background-color: rgb(15, 52, 96);
}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_content">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 26, 46);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_login">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(22, 33, 62);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>40</y>
            <width>150</width>
            <height>150</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/system_monitor/system_monitor.png);
background-size: 10 px;
backgroud-position: center no-repeat;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_user">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>200</y>
            <width>191</width>
            <height>45</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(26, 26, 46);
	border-radius: 3px;
	padding: 15px;
	background-color: rgb(26, 26, 46);
	color: rgb(100, 100, 100);
}

QLineEdit:hover{
	border: 2px solid rgb(100, 100, 100);
}

QLineEdit:focus{
	border: 2px solid rgb(255, 255, 255);
	color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>32</number>
          </property>
          <property name="placeholderText">
           <string>USER</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>250</y>
            <width>191</width>
            <height>45</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(26, 26, 46);
	border-radius: 3px;
	padding: 15px;
	background-color: rgb(26, 26, 46);
	color: rgb(100, 100, 100);
}

QLineEdit:hover{
	border: 2px solid rgb(100, 100, 100);
}

QLineEdit:focus{
	border: 2px solid rgb(255, 255, 255);
	color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>16</number>
          </property>
          <property name="placeholderText">
           <string>PASSWORD</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_title">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>9</y>
            <width>171</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Uroob</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>System Monitor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_user">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>302</y>
            <width>191</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
	border: 2px solid rgb(53, 53, 65);
	border-radius: 5px;
	background-color: rgb(26, 26, 46);
}

QCheckBox::indicator:checked {
	border: 2px solid rgb(53, 53, 65);
	border-radius: 5px;
	background-color: rgb(255, 255, 255);
}

QCheckBox {
	color: rgb(255, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string>SAVE USER</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_connect">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>333</y>
            <width>191</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: 2px solid rgb(26, 26, 46);
	border-radius: 3px;
	background-color: rgb(26, 26, 46);
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	border: 2px solid rgb(100, 100, 100);
	background-color: rgb(15, 52, 96);
}
QPushButton:pressed{
	border: 2px solid rgb(255, 255, 255);
	background-color: rgb(15, 52, 96);
	color: rgb(100, 100, 100);
}</string>
          </property>
          <property name="text">
           <string>CONNECT</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="buttom_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(15, 52, 96);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_credit">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Created by: Oussama talaoui</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="file_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
